ros2_batch_job/packaging.py

```diff
@@ -44,10 +44,10 @@ def build_and_test_and_package(args, job):
     ] + (['--merge-install'] if not args.isolated else []) + \
         args.build_args
 
-    cmake_args = ['" -DBUILD_TESTING=OFF" " --no-warn-unused-cli"']
+    cmake_args = ['-DBUILD_TESTING=OFF --no-warn-unused-cli']
```

Comment:
Shouldn't these two be separate items in the list?

Same below.

Reply:
:+1: updated in https://github.com/ros2/ci/pull/183/commits/5f02ef19e1796c7a45847da2aedec3cb8e169579 and rebased